config/config.exs

```diff
@@ -19,6 +19,9 @@ config :animina, env: Mix.env()
 # Configures the number of stories required for complete registration
 config :animina, :number_of_stories_required_for_complete_registration, 1
 
+# Configures the minimum number of points required to view the 'AI Help' Chat Message in the chat
+config :animina, :minimum_points_required_for_ai_help_in_chat, 19
```

Comment:
Isn't this the wrong approach? We charge 20 points for AI help for the first message, don't we? Where do we charge those 20 points? That place should use a `ai_message_help_price` which gets set in `config.exs` to 20. AND that same variable should be used in `chat_live.ex`.

- One central place to configure the value.
- Name that makes sense for somebody who reads the code for the first time.

Reply:
@MICHAELMUNAVU83 Just a reminder that this is not finished yet.

Reply:
@wintermeyer  , just to get it right would the only change be to change the name of the config?

Reply:
Not only the name. The value too. I understand why you used 19 but for any other coder who is going to read this code in a year 20 would make more sense. So please use 20 and change the other code to accommodate this.

Reply:
@wintermeyer  , I have now made those changes